devlib/target.py

```diff
@@ -1041,6 +1041,12 @@ def adb_reboot_bootloader(self):
     def adb_root(self):
         adb_command(self.adb_name, 'root', timeout=30)
 
+    def usb_charge(self, enable=True, sysfs_path=None):
```

Comment:
I rather prefer the (just discovered) patch by @bjackman ... going to cherry-pick that and refresh this PR ;-)